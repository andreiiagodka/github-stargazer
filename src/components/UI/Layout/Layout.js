import React, { Fragment } from 'react'
import PropTypes from 'prop-types'

import { Card } from 'react-bootstrap'

const Layout = ({ header, body }) => (
  <Fragment>
    <Card.Header>{header}</Card.Header>
    <Card.Body>{body}</Card.Body>
  </Fragment>
)

Layout.propTypes = {
  header: PropTypes.node.isRequired,
  body: PropTypes.node.isRequired
}

export default Layout