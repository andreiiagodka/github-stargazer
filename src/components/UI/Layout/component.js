import React, { Fragment } from 'react'
import PropTypes from 'prop-types'

import { Card } from 'react-bootstrap'

const LayoutComponent = ({ header, body }) => (
  <Fragment>
    <Card.Header>{header}</Card.Header>
    <Card.Body>{body}</Card.Body>
  </Fragment>
)

LayoutComponent.propTypes = {
  header: PropTypes.node.isRequired,
  body: PropTypes.node.isRequired
}

export default LayoutComponent
