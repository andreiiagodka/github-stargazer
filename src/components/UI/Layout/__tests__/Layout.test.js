import React from 'react'
import renderer from 'react-test-renderer'

import Layout from '../Layout'

describe('<Layout /> component', () => {
  it('renders component correctly', () => {
    const props = {
      header: 'Header',
      body: 'Body'
    }
    const tree = renderer.create(<Layout {...props} />).toJSON()
  
    expect(tree).toMatchSnapshot()
  })
})
