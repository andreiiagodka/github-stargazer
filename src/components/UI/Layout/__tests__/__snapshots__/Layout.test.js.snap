// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> component renders component correctly 1`] = `
Array [
  <div
    className="card-header"
  >
    Header
  </div>,
  <div
    className="card-body"
  >
    Body
  </div>,
]
`;
